nservicebus/messaging/delayed-delivery.md

```diff
@@ -26,6 +26,7 @@ Calling `bus.Defer(...)` the handling of the message is deferred by certain amou
 
 Using this mechanism delayed delivery can be achieved by calling `bus.Defer(...)` which can be seen in [Delayed Delivery Sample](/samples/delayed-delivery).
 
+Note: Similar to `bus.SendLocal`, `bus.Defer` will also change the message's reply-to address to the endpoint deferring the message. Calling `bus.Reply` on a deferred message will send the reply to itself.
```

Comment:
@SimonCropp this page doesn't use version switching yet, so this fragment is only about v5. I can take care of it during review.

@timbussmann do we need to add anything about v6?